RELEASE_NOTES.md

```diff
@@ -1,5 +1,13 @@
 # Stream Engine
 
+Development Release 1.6.0 2018-05-11
```

Comment:
should have a # in front to designate the line as a header

Reply:
Done.